migrationmanager/migrators/metrics/migrator.go

```diff
@@ -32,6 +32,26 @@ func (m *MetricsMigrator) Migrate(ctx context.Context) error {
 			return fmt.Errorf("failed to remove TTL from table %s.%s, err: %s", database, clickhousemetricsexporter.TIME_SERIES_TABLE, err)
 		}
 	}
+
+	// TODO(srikanthccv): remove this after a while
+	// Why?
+	// The table shard key had incorrectly been set to (env, temporality, metric_name, fingerprint, unix_milli) instead of (env, temporality, metric_name, fingerprint)
```

Comment:
shard key `env, temporality, metric_name, fingerprint` is equivalent to just `fingerprint` as it has env, temporality, metric_name already in it?

Reply:
Right, they should lead to identical distribution today.